lesson-3/object_finder.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module ObjectFinder
+  private
+
+  def find(array, attr_name, input)
+    raise 'This object already exists' if array.find { |e| e.public_send(attr_name.to_s) == input }
```

Comment:
.to_s is unnecessary

Reply:
I change the parameter form 'name' to name. And this is wron without .to_s.